pinecone/sparse/splade/splade-quora.ipynb

```diff
@@ -0,0 +1,2439 @@
+{
```

Comment:
<blockquote>Unlike dense embeddings, which can be difficult to interpret, sparse embeddings map to tokens for easier interpretability.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>